bottlecap/src/metrics/dogstatsd.rs

```diff
@@ -21,16 +42,45 @@ impl DogStatsD {
                     msg,
                     src
                 );
-                let dummy_tags = vec!["tagA:valueA".to_string(), "tagB:valueB".to_string()];
-                let metric_event = MetricEvent::new("metric_name".to_string(), 1.0, dummy_tags);
+                let parsed_metric = match Metric::parse(msg) {
+                    Ok(parsed_metric) => {
+                        log::info!("parsed metric: {:?}", parsed_metric);
```

Comment:
In practice you will probably want to move this down to `debug` at some point, although I see the utility now.